### PR TITLE
website_slides: Do not display navigation buttons if slide has only one page

### PR DESCRIPTION
**Pull request by rga-odoo**
task: https://www.odoo.com/web#id=12790&view_type=form&model=project.task&menu_id=3942&action=327
pad: https://pad.odoo.com/p/r.0IdACyLDfChHINKW

### CODE REVIEW

File: addons/website_slides/static/src/js/slides_embed.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ $(document).ready(function () {
                 this.$('canvas').show();
                 this.$('#page_count').text(this.pdf_viewer.pdf_page_total);
                 this.$('#PDFViewerLoader').hide();
+                if (this.pdf_viewer.pdf_page_total > 1) {
+                    $('.slide_nav_button').toggleClass('hide');
```
Comment (rga-odoo) [inline addons/website_slides/static/src/js/slides_embed.js]: $('.slide_nav_button') ==> this.$('.slide_nav_button')

Comment (rsi-odoo) [inline addons/website_slides/static/src/js/slides_embed.js]: @rga-odoo Done

File: addons/website_slides/view/website_slides_embed.xml [hunk 2]
```diff
@@ -187,22 +187,22 @@
                                 <div class="col-lg-8 col-md-8 col-sm-8 col-xs-8">
                                     <div class="col-lg-4 col-md-4 col-sm-4 col-xs-4 text-center"></div>
                                     <div class="col-lg-1 col-md-1 col-sm-1 col-xs-1 text-center">
-                                        <span id="first" title="First slide">
+                                        <span id="first" class="hide slide_nav_button" title="First slide">
```
Comment (jem-odoo) [inline addons/website_slides/view/website_slides_embed.xml]: only add the class `o_slide_navigation_buttons` on the parent div (col-lg-8 col-md-8 col-sm-8 col-xs-8), and apply the `hide()`method if the number of page is 1.
Maybe you will have to add the `pull-right text-right` on the parent of  'download' and 'full screen' buttons to keep them on the right side.

Comment (rsi-odoo) [inline addons/website_slides/view/website_slides_embed.xml]: @jem-odoo If we do this, the ```nav_buttons``` are first seen and then hide that's why I think we have to use the ```.hide``` class in div and then we have can show the ```div``` using ```show()``` method. Is it better for you ???

**Pull request merged**